pkg/inttest/http.go

```diff
@@ -171,6 +171,41 @@ func (hc *HTTPClient) PostJSON(t *testing.T, path string, requestBody io.Reader,
 	require.NoError(t, err, errMsg+": failed to unmarshal response body")
 }
 
+func (hc *HTTPClient) SignIn(t *testing.T, email, password string) (*http.Cookie, *http.Cookie) {
+	t.Helper()
+
+	req := hc.NewRequest(t, http.MethodPost, "/tokens", nil, WithHeader("Content-Type", "application/json"), WithBasicAuth(email, password))
```

Comment:
Why do you pass `Content-Type: application/json` in this post? There is no body right 🤔

Reply:
We only accept those two types of content.

https://github.com/dhis2-sre/im-manager/blob/master/internal/handler/dataBinder.go#L10

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type
> In requests, (such as [POST](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST) or [PUT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT)), the client tells the server what type of data is actually sent. 

You do not send any data, so this header should not be needed, right?